CRM/Contribute/WorkflowMessage/Contribution/BasicContribution.php

```diff
@@ -47,6 +60,37 @@ public function build(array &$example): void {
     $messageTemplate = new $workFlow['class']();
     $this->addExampleData($messageTemplate, $example);
     $example['data'] = $this->toArray($messageTemplate);
+    switch ($example['workflow']) {
+      case 'contribution_online_receipt':
+        $example['asserts'] = [
+          'default' => [
+            ['for' => 'subject', 'regex' => '/Receipt - FIXME Contribution Title - Barbara Johnson/'],
+            ['for' => 'html', 'regex' => '/table id="crm-event_receipt"/'],
+            ['for' => 'html', 'regex' => '/Dear Barb,/'],
+          ],
+        ];
+        break;
+
+      // This does not yet get hit - placeholder
+      case 'contribution_offline_receipt':
+        $example['asserts'] = [
+          'default' => [
+            ['for' => 'subject', 'regex' => '/Contribution Receipt - Barbara Johnson/'],
+            ['for' => 'text', 'regex' => '/Transaction ID: 123/'],
+          ],
+        ];
+        break;
+
+      // This does not yet get hit - placeholder
+      case 'contribution_invoice_receipt':
+        $example['asserts'] = [
+          'default' => [
+            ['for' => 'subject', 'regex' => '/Invoice - Barbara Johnso/'],
```

Comment:
There's an "n" missing at end but the comment says not in use yet anyway.

Reply:
yeah - for some reason Tim used that pattern in his version of this & I .. copied & pasted